fitbenchmarking/cli/main.py

```diff
@@ -134,6 +134,15 @@ def get_parser():
                        help="Use this option if you have decided not to"
                        "create plots during runtime.")
 
+    group.add_argument('--results_browser', action='store_true',
+                       help="Use this option if you have decided to"
+                       "open a browser window to show the results"
+                       "of a fit benchmark.")
+    group.add_argument('--dont_open_browser', action='store_true',
```

Comment:
I think the pair should match, i.e. something like:
- "open_browser" and "dont_open_browser"
- "results_browser" and "no_results_browser"
- "open_results" and "dont_open_results"